### PR TITLE
Add e2e test for v1beta1 BackendConfig features

### PR DESCRIPTION
**Pull request by skmatti**

### CODE REVIEW

File: cmd/e2e-test/security_policy_v1beta1_test.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+	ctx := context.Background()
+	t.Parallel()
+
+	Framework.RunWithSandbox("Security Policy Enable (backendconfig v1beta1)", t, func(t *testing.T, s *e2e.Sandbox) {
+		policies := []*computebeta.SecurityPolicy{
+			buildPolicyAllowAll(fmt.Sprintf("enable-test-allow-all-%s", s.Namespace)),
+		}
+		defer func() {
+			if err := cleanupSecurityPolicies(ctx, t, Framework.Cloud, policies); err != nil {
+				t.Errorf("cleanupSecurityPolicies(...) =  %v, want nil", err)
```
Comment (bowei) [inline cmd/e2e-test/security_policy_v1beta1_test.go:46]: t.Logf

We should not Errorf in the defer()

Comment (bowei) [reply cmd/e2e-test/security_policy_v1beta1_test.go:46]: Is it not possible to write a generic test with all of the fields of BackendConfig so we don't need separate tests for v1beta1 on those fields?

Comment (skmatti) [reply cmd/e2e-test/security_policy_v1beta1_test.go]: Done

**Pull request closed without merging**